python/lsst/ip/diffim/detectAndMeasure.py

```diff
@@ -472,6 +472,10 @@ def deblend(footprints):
             sources['parent'] = 0
             return sources.copy(deep=True)
 
+        # Clear existing deblender mask planes, if present
+        if "NOT_DEBLENDED" in difference.mask.getMaskPlaneDict().keys():
+            mask = difference.mask
+            mask &= ~(mask.getPlaneBitMask("NOT_DEBLENDED"))
```

Comment:
I think this should happen at the same place the clearing of the detected planes happens, at line 337. You can just add it to the ORed bits being cleared there.

Reply:
We aren't guaranteed to have `NOT_DEBLENDED` present, so we still need to have this check. I can consolidate this with clearing the `DETECTED` mask.